app/services/mapping_migration_service.rb

```diff
@@ -0,0 +1,90 @@
+class MappingMigrationService
+  attr_reader :integration_name, :rtp_id, :template_file, :templates_dir, :upload_integrations
+
+  def initialize
+    @templates_dir = Configuration.paths_templates_plugins
+    @upload_integrations = Dradis::Plugins::with_feature(:upload)
+  end
+
+  def call
+    upload_integrations.each do |integration|
+      @integration_name = integration.plugin_name.to_s
+
+      template_files.each do |template_file|
+        @template_file = template_file
+
+        if defined?(Dradis::Pro)
```

Comment:
I think we can improve the parity here with Pro. What about this approach instead:
```
rtp_ids = defined?(Dradis::Pro) ? ReportTemplateProperties.pluck(:id) : [nil]
rtp_ids.each do |rtp_id|
  migrate
  ...
```

This will remove the need for a pro-specific change and migrate_pro entirely. Thoughts?